src/networking/NetworkService.js

```diff
@@ -1,11 +1,20 @@
 import axios from 'axios';
-import { baseURL, headers } from '@/networking/config';
+import {
+  baseURL as apiBaseUrl,
+  headers,
+  movieDBAPIUrl,
+  movieDBBearerToken,
+} from '@/networking/config';
 import { resInterceptor } from '@/networking/interceptors';
 
 export class NetworkService {
-  constructor() {
-    this.client = axios.create({ baseURL, headers });
+  constructor(_paramConfig = {}) {
+    const defaultConfig = { baseURL: apiBaseUrl, headers };
+    const { bearerToken, ...paramConfig } = _paramConfig;
+    const config = { ...defaultConfig, ...paramConfig };
+    this.client = axios.create(config);
     this.client.interceptors.response.use(resInterceptor.onFulfill, resInterceptor.onReject);
+    if (bearerToken) this.setAccessToken(bearerToken);
```

Comment:
If we don't do it this way, `circleci lint` will fail.

```suggestion
    if (bearerToken) {
          this.setAccessToken(bearerToken);
     }
```